src/components/User/KycRequests/components/PendingRequestActions.vue

```diff
@@ -0,0 +1,160 @@
+<template>
```

Comment:
actually works with decisions of the queue but the path of the file is confusing

Reply:
Extracted to `/queue` folder